addons/html_editor/static/src/main/media/image_crop.js

```diff
@@ -154,7 +154,8 @@ export class ImageCrop extends Component {
         }
 
         this.cropperWrapper.el.style.left = `${offset.left}px`;
-        this.cropperWrapper.el.style.top = `${offset.top}px`;
+        this.cropperWrapper.el.style.top =
+            offset.top < 0 ? `${window.innerHeight * 0.1}px` : `${offset.top}px`;
```

Comment:
I think currently the crop container is positioned to the bottom rather than than the top, see https://github.com/odoo/odoo/blob/master/addons/html_editor/static/src/main/media/image_crop.js#L229C1-L231C29 However since https://github.com/odoo/odoo/pull/114731 the crop buttons are always visible on screen, we no longer need to position it at the bottom; we need to scroll the image to its top instead.